bite-size-notes/week-6-c-state-estimation-intro.md

```diff
@@ -0,0 +1,193 @@
+# Deterministic State Estimation
```

Comment:
Overall, these notes exactly covers the theory behind the state estimation, the Luenberger Observer. Let me know if you have any questions. I am Guanyu Ren.

Reply:
Thank you for your comments!